tests/JsonLd/ContextBuilderTest.php

```diff
@@ -141,32 +142,56 @@ public function testAnonymousResourceContext()
             '@context' => [
                 '@vocab' => '#',
                 'hydra' => 'http://www.w3.org/ns/hydra/core#',
-                'dummyPropertyA' => $iri.'/dummyPropertyA',
+                'dummyPropertyA' => 'Dummy/dummyPropertyA',
             ],
             '@id' => $iri,
+            '@type' => 'Dummy',
```

Comment:
This test might be confusing because `Dummy` is indeed annotated with `@ApiResource`? :stuck_out_tongue_closed_eyes: